app/assets/javascripts/test/snapshots/public/test-bundle/test/backend-snapshot-tests/timetracking.e2e.js.md

```diff
@@ -21,10 +21,10 @@ Generated by [AVA](https://ava.li).
       {
         activeInstances: 1,
         finishedInstances: 0,
-        openInstances: 19,
+        openInstances: 39,
         paused: false,
         projectName: 'Test_Project',
-        totalInstances: 20,
+        totalInstances: 40,
```

Comment:
do you know what happened here? is this expected?

Reply:
@fm3 No I don't know and it's not expected